mobile/src/main/java/ch/epfl/sdp/mobile/state/StatefulFollowingScreen.kt

```diff
@@ -114,8 +148,23 @@ private class SnapshotSocialScreenState(
       }
     }
   }
+}
 
-  override fun onPersonClick(person: ProfileAdapter) {
-    onPersonItemClick(person)
+/**
+ * Reduces a [Flow] of [Interaction] to determine whether the flow represents a currently focused
+ * element. This is a [Flow]-based equivalent to [MutableInteractionSource]'s
+ * `collectIsFocusedAsState()`.
+ *
+ * @return the [Flow] of [Interaction] that is reduced.
```

Comment:
[Question] I don't understand this doc. The fun returns a Flow that emits true iff the state is focused by reducing the interactions but it doesn't return the reduced flow itself as opposed to what the doc says. Or am I missing something ?

Reply:
Okay, apparently I had written `@return` instead of `@receiver` in the docs. This is fixed in 8d1c92e2a3b004f0bd461ea49dcd1025fec7d290.